seatunnel-connectors-v2/connector-file/connector-file-base/src/main/java/org/apache/seatunnel/connectors/seatunnel/file/config/FileFormat.java

```diff
@@ -55,7 +55,10 @@ public ReadStrategy getReadStrategy() {
         public ReadStrategy getReadStrategy() {
             return new JsonReadStrategy();
         }
-    };
+    },
+    EXCEL("xlsx")
+    ;
+
 
```

Comment:
hi, you should add a `ExcelReadStrategy`

Reply:
Isn't `ExcelReadStrategy` added by `Excel Source`?